platform-hub-api/app/models/audit.rb

```diff
@@ -15,6 +15,7 @@ class Audit < ApplicationRecord
   scope :ascending, -> { reorder(version: :asc) }
   scope :descending, -> { reorder(version: :desc) }
   scope :by_action, -> (action) { where(action: action) }
+  scope :by_actions, -> (actions) { where("action IN (?)", actions) }
```

Comment:
I think you can just use the exact same `where` condition as the scope above (i.e. just `where(action: actions)` and AR will automagically convert it into an `IN` query :)